src/Renderer/Draw/QuadraticPath.ts

```diff
@@ -9,20 +9,37 @@ export const QuadraticPath = (
   cornerRadius: number,
   strokeWidth: number,
   color: string,
-  centerPoint: number = 0.5
+  centerPoint: number = 0.5,
+  circularReference: boolean
 ): void => {
-  const centerX = LinkUtils.calculateLinkXCenterMath(x1, x2, centerPoint);
-  let ydiff = Math.abs(y2 - y1);
-  let cr = ydiff > cornerRadius * 2 ? cornerRadius : Math.floor(ydiff / 2.0);
-  let crx = x2 > x1 ? cr : -cr;
-  let cry = y2 > y1 ? cr : -cr;
   context.beginPath();
   context.moveTo(x1, y1);
-  context.lineTo(centerX - crx, y1);
-  context.arcTo(centerX, y1, centerX, y1 + cry, cr);
-  context.lineTo(centerX, y2 - cry);
-  context.arcTo(centerX, y2, centerX + crx, y2, cr);
-  context.lineTo(x2, y2);
+    console.log(x1, y1);
+  if (circularReference) {
+    let hv = 150; // horizontal vector
```

Comment:
1) Use `const` instead of `let` for constant values
2) Extract those values out of the function and name them properly as constants (e.g. `NODE_CR_HORIZONTAL_VECTOR` etc.)